apps/etl/transformer.py

```diff
@@ -0,0 +1,175 @@
+import logging
+import time
+
+from celery import shared_task
+from celery.result import AsyncResult
+from pystac_monty.sources.gdacs import (
+    GDACSDataSource,
+    GDACSDataSourceType,
+    GDACSTransformer,
+)
+
+from apps.etl.models import ExtractionData, GdacsTransformation
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def transform_event_data(data):
+    logger.info("Trandformation started for event data")
+
+    gdacs_instance = ExtractionData.objects.get(id=data["extraction_id"])
+
+    data_file_path = gdacs_instance.resp_data.path  # Absolute file path
+
+    try:
+        with open(data_file_path, "r") as file:
+            data = file.read()
+    except FileNotFoundError:
+        logger.error(f"File not found: {data_file_path}")
+        raise
+    except IOError as e:
+        logger.error(f"I/O error while reading file: {str(e)}")
+        raise
```

Comment:
User direct `gdacs_instance.resp_data.read()`, this is handled by the backend storage class.
https://docs.djangoproject.com/en/5.1/ref/files/storage/